### ISSUE TITLE
PCA group

### ISSUE THREAD

**Issue by drlemmus** [MRC-LMB-ComputationalStructuralBiology/monomers#54]
Currently PCA is marked as non-polymer. Should this not be a P-peptide, or does the double bonded oxygen mess things up?

**Comment by keitaroyam** [comment]
Yes, because of that, N is sp2 and therefore this cannot be P-peptide. Probably we should define the link for this (is it always at the N-terminal?)

**Comment by drlemmus** [comment]
Biologically, yes it should be N-terminal. Because it can bind from its acidic group, which is normal, I guess It could be a 'normal' peptide.

**Comment by keitaroyam** [comment]
Currently, a monomer could be a peptide (or P-/M-peptide) only when it's linkable from both sides. This may be too strict - but otherwise it may cause accidental links from the other side. I need to think about this more carefully.
